.github/ISSUE_TEMPLATE/testplan.md

```diff
@@ -1292,12 +1292,18 @@ tsh bench web sessions --max=5000 --web user ls
 - Warnings/Errors (test by applying [this patch](https://gist.github.com/ibeckermayer/7591333275e87ad0d7afa028a7bb54cb))
   - [ ] Induce the backend to send a TDP Notification of severity warning (1), confirm that a warning is logged in the warning dropdown
   - [ ] Induce the backend to send a TDP Notification of severity error (2), confirm that session is terminated and error popup is shown
-  - [ ] Induce the backend to send a TDP Error, confirm that session is terminated and error popup is shown (confirms backwards compatibility w/ older w_d_s starting in Teleport 12)
```

Comment:
Why remove this?

Reply:
My initial thinking was that we no longer actually send any `tdp.Error`s, they've been replaced with tdp.Notification (same reason why I'd ripped out `tdp.Error` generally, before you pointed out the recording problem). I can change this to confirming that a recording with this works.

Reply:
064ec4f34463d64db4e85283652d7cc1380947af